src/desktop/src/ui/components/input/Text.js

```diff
@@ -47,6 +48,7 @@ export default class Text extends React.PureComponent {
                         type="text"
                         value={value}
                         onChange={(e) => onChange(e.target.value)}
+                        maxLength={MAX_MESSAGE_LENGTH}
```

Comment:
This will limit all text fields to 1093 characters. Instead, we should pass a `maxLength` prop which will be used similar to the `label` prop

Reply:
I'm not sure how that should look, should it be like
```
...   
/** Limit max text length */
maxLength: PropTypes.integer,
...
const { value, label, maxLength, onChange, disabled } = this.props;
```
?

Reply:
Yep, that's correct